glpi-project/glpi
# Remove compiled mo from repository

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

1. Check that mo files exists and are up to date (same logic as for composer/npm dependencies).
2. Move mo compilation script into console to ensure perl is not required.
3. Remove compiled mo files from repository.

On my machine, check seems to take less than 1ms. I guess file stats are cached on OS side.
